From: Adam Gandelman <adamg@ubuntu.com>
Date: Wed May 29 12:13:38 PDT 2013
Subject: Avoid Cheetah dependency.
Ubuntu-bug: https://bugs.launchpad.net/ubuntu/+source/cheetah/+bug/1183634
Until Cheetah's dependencies have been MIR'd into Main, avoid it
being listed in requirements.txt and requiring it as a runtime
dependency.


Index: nova-src/requirements.txt
===================================================================
--- nova-src.orig/requirements.txt	2013-06-19 18:03:08.469447733 -0700
+++ nova-src/requirements.txt	2013-06-19 18:03:08.465447733 -0700
@@ -1,7 +1,6 @@
 d2to1>=0.2.10,<0.3
 pbr>=0.5.16,<0.6
 SQLAlchemy>=0.7.8,<0.7.99
-Cheetah>=2.4.4
 amqplib>=0.6.1
 anyjson>=0.2.4
 argparse
